stl/inc/expected

```diff
@@ -0,0 +1,1022 @@
+        } else {
+            _STD construct_at(_STD addressof(_Unexpected), _Other._Unexpected);
+        }
+    }
+
+    // clang-format off
+    expected(const expected& _Other) requires
+        is_trivially_copy_constructible_v<_Ty> && is_trivially_copy_constructible_v<_Err> = default;
+
+    constexpr expected(expected&& _Other) noexcept(
+        is_nothrow_move_constructible_v<_Ty>&& is_nothrow_move_constructible_v<_Err>) //
+        requires (!is_trivially_move_constructible_v<_Ty> && !is_trivially_move_constructible_v<_Err> //
+                 && is_move_constructible_v<_Ty> && is_move_constructible_v<_Err>)
+        : _Has_value(_Other._Has_value) {
+        // clang-format on
+        if (_Has_value) {
+            _STD construct_at(_STD addressof(_Value), _STD move(_Other._Value));
+        } else {
+            _STD construct_at(_STD addressof(_Unexpected), _STD move(_Other._Unexpected));
+        }
+    };
+
+    // clang-format off
+    expected(expected&&) requires
+        is_trivially_move_constructible_v<_Ty> && is_trivially_move_constructible_v<_Err> = default;
+    // clang-format on
+
+    template <class _Uty, class _UErr>
+    static constexpr bool _Is_not_convertible = !is_constructible_v<_Ty, expected<_Uty, _UErr>&> //
+                                             && !is_constructible_v<_Ty, expected<_Uty, _UErr>> //
+                                             && !is_constructible_v<_Ty, const expected<_Uty, _UErr>&> //
+                                             && !is_constructible_v<_Ty, const expected<_Uty, _UErr>> //
+                                             && !is_convertible_v<expected<_Uty, _UErr>&, _Ty> //
+                                             && !is_convertible_v<expected<_Uty, _UErr>&&, _Ty> //
+                                             && !is_convertible_v<const expected<_Uty, _UErr>&, _Ty> //
+                                             && !is_convertible_v<const expected<_Uty, _UErr>&&, _Ty> //
+                                             && !is_constructible_v<unexpected<_Err>, expected<_Uty, _UErr>&> //
+                                             && !is_constructible_v<unexpected<_Err>, expected<_Uty, _UErr>> //
+                                             && !is_constructible_v<unexpected<_Err>, const expected<_Uty, _UErr>&> //
+                                             && !is_constructible_v<unexpected<_Err>, const expected<_Uty, _UErr>>;
+
+    template <class _Uty, class _UErr>
+        requires is_constructible_v<_Ty, const _Uty&> && is_constructible_v<_Err, const _UErr&> //
+            && _Is_not_convertible<_Uty, _UErr>
+    constexpr explicit(!is_convertible_v<const _Uty&, _Ty> || !is_convertible_v<const _UErr&, _Err>)
+        expected(const expected<_Uty, _UErr>& _Other) noexcept(is_nothrow_constructible_v<_Ty, const _Uty&>&&
+                is_nothrow_constructible_v<_Err, const _UErr&>) /* strengthened */

```

Comment:
Nitpick: We use the C-style comment `/* strengthened */` only when it occurs in the middle of a line. At the end of a line, we should use the C++-style `// strengthened`.

Occurs below.